### PR TITLE
Add starter code

### PR DESCRIPTION
**Pull request by connorjayr**
Adds starter code for the web server, using `actix-web` as the server framework. Also adds SPA routing to the frontend and integrates ESLint and Prettier for code style enforcement.

### CODE REVIEW

File: server/src/main.rs [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+use actix_web::{middleware::Logger, App, HttpServer};
+
+#[actix_web::main]
+async fn main() -> io::Result<()> {
+    env_logger::init();
+
+    HttpServer::new(|| {
+        App::new()
+            .wrap(Logger::default())
+            .service(routes::login::handler)
```
Comment (jputlock) [inline server/src/main.rs:15]: This is fine for now -- we may want to move responsibility of configuring groups of routes (such as a group containing `login` and `logout`) elsewhere using `.configure()`

Comment (jputlock) [reply server/src/main.rs:15]: Also if we want to move them into an `auth` scope, we can use `web::scope()`

**Review by jputlock — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,46 +1,3 @@
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+Sust is a modern, role-based murder mystery game built using Rust and React.
```
Comment (jputlock) [inline README.md:3]: Please add a note that `cargo-watch` is required.

**Review by jputlock — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jputlock) [reply README.md:3]: Also a note about the correct `node` version.

Comment (connorjayr) [reply README.md:3]: Added both in https://github.com/connorjayr/sust/pull/1/commits/d8a12d4d9a91afd8b9d04e5c72fb81b382981dfc.

File: server/src/main.rs (hunk 1 above)
Comment (connorjayr) [reply server/src/main.rs:15]: Agree with both of the above. I think routes with shared prefixes can even create a `Scope` within the module itself, and then the caller does not need to worry about any of the details.

**Review by jputlock — APPROVED** [review]
LGTM 👍

**Pull request merged**